Radzen.Blazor/themes/components/blazor/_tabs.scss

```diff
@@ -17,193 +17,263 @@ $tabs-icon-margin: 0 !default;
-          &:not(.rz-state-disabled) {
+        li {
+            border-left-width: 2px;
+            border-right-color: $tabs-tab-background-color;
+            border-radius: $tabs-border-radius 0 0 $tabs-border-radius;
+
+            &:hover {
+                &:not(.rz-tabview-selected) {
+                    &:not(.rz-state-disabled) {
+                        border-left-color: $tabs-tab-selected-top-border-color;
+                    }
+                }
+            }
+        }
+
+        .rz-tabview-selected {
+            border-right-color: $tabs-background-color;
+            margin-right: -1px;
             border-left-color: $tabs-tab-selected-top-border-color;
-          }
         }
-      }
     }
-  
-    .rz-tabview-selected {
-      border-right-color: $tabs-background-color;
-      margin-right: -1px;
-      border-left-color: $tabs-tab-selected-top-border-color;
-    }
-  }
 
-  .rz-tabview-right > & {
-    flex-direction: column;
+    .rz-tabview-right > & {
+        flex-direction: column;
 
-    li {
-      border-right-width: 2px;
-      border-left-color: $tabs-tab-background-color;
-      border-radius: 0 $tabs-border-radius $tabs-border-radius 0;
-  
-      &:hover {
-        &:not(.rz-tabview-selected) {
-          &:not(.rz-state-disabled) {
+        li {
+            border-right-width: 2px;
+            border-left-color: $tabs-tab-background-color;
+            border-radius: 0 $tabs-border-radius $tabs-border-radius 0;
+
+            &:hover {
+                &:not(.rz-tabview-selected) {
+                    &:not(.rz-state-disabled) {
+                        border-right-color: $tabs-tab-selected-top-border-color;
+                    }
+                }
+            }
+        }
+
+        .rz-tabview-selected {
+            border-left-color: $tabs-background-color;
+            margin-left: -1px;
+            padding-left: 1px;
             border-right-color: $tabs-tab-selected-top-border-color;
-          }
         }
-      }
-    }
-  
-    .rz-tabview-selected {
-      border-left-color: $tabs-background-color;
-      margin-left: -1px;
-      padding-left: 1px;
-      border-right-color: $tabs-tab-selected-top-border-color;
     }
-  }
 }
 
 .rz-tabview-panels {
-  background-color: $tabs-background-color;
-  border: $tabs-border;
-  box-shadow: $tabs-shadow;
-  flex: 1;
-  overflow: auto;
-
-  .rz-tabview-top > & {
-    border-radius: 0 0 $border-radius $border-radius;
-  }
-  .rz-tabview-bottom > & {
-    border-radius: $border-radius $border-radius 0 0;
-  }
-  .rz-tabview-left > & {
-    border-radius: 0 $border-radius $border-radius 0;
-  }
-  .rz-tabview-right > & {
-    border-radius: $border-radius 0 0 $border-radius;
-  }
+    background-color: $tabs-background-color;
+    border: $tabs-border;
+    box-shadow: $tabs-shadow;
+    flex: 1;
+    overflow: auto;
+
+    .rz-tabview-top > & {
+        border-radius: 0 0 $border-radius $border-radius;
+    }
+
+    .rz-tabview-top-right > & {
```

Comment:
We don't need this rule as it will be inherited from the `.rz-tabview-top` rule above. Same applies to the `bottom-right` rule below.